<launch>
    <arg name="show_img" default="false" />
    <arg name="fps" default = "30" />
    <arg name="resolution" default = "720"/>
    <arg name="auto_expose" default = "false"/>
    <arg name="expose_time_us" default = "15000"/>
    <arg name="iso" default = "200"/>
    <arg name="image_info" default= "true"/>
    <arg name="auto_awb" default = "true"/>
    <arg name="awb_value" default = "4000"/>
    <arg name="ros_defined_freq" default = "false"/>
    <arg name="enable_compressed_image" default = "true"/>
    <arg name="enable_assemble_image" default = "true"/>

    <node name="oakcam_ffc_4p_ros" pkg="oakcam_ffc_4p_ros" type="oakcam_ffc_4p_ros_node" output="screen">
        <param name="show_img" value="$(arg show_img)" />
        <param name="fps" value="$(arg fps)" />
        <param name="resolution" value = "$(arg resolution)"/>
        <param name="auto_expose" value = "$(arg auto_expose)"/>
        <param name="expose_time_us" value = "$(arg expose_time_us)"/>
        <param name="iso" value = "$(arg iso)"/>
        <param name="image_info" value = "$(arg image_info)"/>
        <param name="auto_awb" value = "$(arg auto_awb)"/>
        <param name="awb_value" value = "$(arg awb_value)"/>
        <param name="ros_defined_freq" value = "$(arg ros_defined_freq)"/>
        <param name="enable_compressed_image" value = "$(arg enable_compressed_image)"/>
        <param name="enable_assemble_image" value= "$(arg enable_assemble_image)"/>
        <!-- <param name="cap_device" value="$(arg cap_device)" />
        <param name="width" value="$(arg width)" />
        <param name="height" value="$(arg height)" />
        <param name="fps" value="$(arg fps)" />
        <param name="enable_color" value="$(arg enable_color)" />
        <param name="publish_splited" value="$(arg publish_splited)" />
        <param name="show" value="$(arg show)" />
        <param name="raw8" value="true" />
        <param name="is_sync" value="$(arg is_sync)" />
        <param name="exposure" value="$(arg exposure)" />
        <param name="gain" value="$(arg gain)" />
        <param name="print_clearness" value="$(arg print_clearness)" /> -->
    </node>
</launch>
